resources/lib/cabertoss.py

```
@@ -88,17 +88,26 @@ def gather_log_files():
+    - Creates a destination directory at Store.destination_path named "<hostname>_Kodi_Logs_<YYYY-MM-DD_HH-MM-SS>".
+    - For entries with type 'log' or 'oldlog', reads the source, sanitizes the content with clean_log() (because these paths may be URLs with embedded user/password details), and writes the sanitized content to a file with the same basename in the destination folder.
+    - For other types (e.g., crash logs), copies the source file to the destination folder unchanged.
+
+    Parameters:
+        log_files (List): list of log descriptors [type, path] to copy.
+
+    Returns:
+        bool: True if files were successfully copied, False otherwise.
     """
     if not log_files:
         Logger.error(LANGUAGE(32025))
```

Ruff (0.13.1)

`LANGUAGE` may be undefined, or defined from star imports

(F405)